src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const rentPeriod = days;
+  const shortTerm = 3;
+  const LongTerm = 7;
+  let discount = 50;
+
+  if (rentPeriod < LongTerm) {
+    discount = 20;
```

Comment:
Don't use literals in expressions (HARDCODE). Instead use constants explaining this values
![image](https://github.com/user-attachments/assets/0395c95e-a8e9-4981-a5f5-36e2b3fb8cdd)

Reply:
Not fixed